### DISCUSSION TITLE
غیر فعال بدون wg-quick@warp  در لاگ سیستم

### DISCUSSION THREAD

**Discussion by skydarklord** [hiddify/Hiddify-Manager#3245] [Bug باگ: please use issue]
با عرض سلام خدمت دوستان گرامی 
چند وقت هست که در بخش لاگ سیستم طبق عکسی که آپلود کردم گزینه wg-quick@warp    / failed رو مشاهده میکنم .
میخواستم بدونم مشکل از کدوم بخش هست .

با تشکر

![image](https://github.com/hiddify/hiddify-config/assets/132658758/c30ebc7f-eb8c-4599-ac1c-d61650976510)

**Comment by skydarklord** [comment]
سلام دوستان . کسی راهنمایی نمیکنه چطور این ارور رو رفع کنم ؟

**Reply by DanielNoohi** [reply]
منم همین مشکل رو خوردم ریست، ری اینستال، او اس ریست، اپلای کانفیگ هیچی جواب نیست

**Reply by DanielNoohi** [reply]
یه بار از اول دستور نصب رو زدم از اول نصب کرد بدون اینکه کاربر یا لینکی رو حذف بکنه درست شد

**Reply by skydarklord** [reply]
سلام ممنون بابت راهنمایی . پس دوباره نصب کنم هیچی مشکلی تو کاربرها و کانفیگها پیش نمیاد ؟ میترسم انجام بدم و همه چیز حذف بشه مشکله که دوباره از اول کاربر تعریف کنم .

**Reply by amirrezaalasti** [reply]
سلام من این کار رو کردم درست نشد @DanielNoohi
